build.sbt

```diff
@@ -53,19 +54,32 @@ sourceGenerators in Compile ++= Seq(
 
 // Lib dependencies
 val sonarVersion = "7.7"
+val circe = "0.11.0"
+val http4s = "0.20.0-M4"
 libraryDependencies ++= List(
-  "org.sonarsource.sonarqube" % "sonar-plugin-api" % sonarVersion % Provided,
-  "org.slf4j"                 % "slf4j-api"        % "1.7.26" % Provided,
-  "org.typelevel"             %% "cats-core"       % "1.6.0",
-  "org.scalariform"           %% "scalariform"     % "0.2.8",
-  "org.scalastyle"            %% "scalastyle"      % "1.0.0",
-  "org.scala-lang.modules"    %% "scala-xml"       % "1.2.0",
-  "org.scalatest"             %% "scalatest"       % "3.0.7" % Test,
-  "org.mockito"               %% "mockito-scala"   % "1.3.1" % Test
+  "org.sonarsource.sonarqube" % "sonar-plugin-api"      % sonarVersion % Provided,
+  "org.slf4j"                 % "slf4j-api"             % "1.7.26" % Provided,
+  "org.typelevel"             %% "cats-core"            % "1.6.0",
+  "org.typelevel"             %% "cats-effect"          % "1.1.0",
+  "org.typelevel"             %% "mouse"                % "0.20",
+  "io.circe"                  %% "circe-core"           % circe,
+  "io.circe"                  %% "circe-generic"        % circe,
+  "io.circe"                  %% "circe-generic-extras" % circe,
+  "org.http4s"                %% "http4s-blaze-client"  % http4s,
+  "org.http4s"                %% "http4s-circe"         % http4s,
```

Comment:
_Yeih_, **Typelevel** friends 😸

Reply:
[**http4s** v`0.20.0`](https://http4s.org/v0.20/) was released a couple of days ago :)